CefSharp.BrowserSubprocess.Core/Async/JavascriptAsyncObjectWrapper.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright � 2010-2015 The CefSharp Project. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#include "stdafx.h"
+#include "JavascriptCallbackRegistry.h"
+#include "JavascriptAsyncObjectWrapper.h"
+#include "JavascriptAsyncMethodWrapper.h"
+
+using namespace System::Linq;
+
+namespace CefSharp
+{
+    namespace Internals
+    {
+        namespace Async
+        {
+            void JavascriptAsyncObjectWrapper::Bind(const CefRefPtr<CefV8Value> &value)
```

Comment:
I think passing in the `CefV8Value` is better, originally there was a need to store a reference, don't think it's required anymore.

So I'm thinking we go with that for all the `Bind` methods?

I updated `master` accordingly yesterday https://github.com/cefsharp/CefSharp/commit/3d19b3a587e364b1501bfc3423185fc132047a79